_dasldemos/proof-troubleshooting.md

```diff
@@ -0,0 +1,116 @@
+    - **Missing closing bracket in workflow/task**: let’s say that in all of my development enthusiasm, I forgot to put a closing bracket on one of the sections of my workflow and/or task. The resulting validation message should say something to the effect of `ERROR: Unexpected symbol (line ##, col #)… Expected rbrace, got <something else>`. If you navigate to the line in question, the missing closing bracket should be a few lines above that.
+    
+    <img src="/dasldemos/assets/proof_ts_missing_bracket.png" alt="missing_bracket" width="650"/>
+    
+    - If you see any unique messages besides these ones, please send us a screenshot at [wilds@fredhutch.org](mailto:wilds@fredhutch.org) or schedule a Research Computing DHC with the DaSL staff. We’ll help work through it with you and add it to the list!
+
+## Task Level Issues
+
+### Identify which task failed
+
+- Start by navigating to [the “Track Jobs” tab of PROOF](/dasldemos/proof-how-to#tracking-your-submitted-jobs) and click “Update View” to produce a list of recent jobs run via PROOF.
+- Scroll down to the “Workflows Run” table and click on the job in question. This will populate the “Workflow Specific Job Information” tables further down in the browser.
+- Scroll down to the “Job List” table and identify which specific task(s) failed within the workflow in question. This will narrow down your issue to a particular task and the table itself will provide information that will be useful in your troubleshooting endeavors.
+
+![job_list_failed](/dasldemos/assets/proof_ts_job_list_failed_task.png)
+
+### Identify why the task failed
+
+- If you scroll down even further, you’ll notice a “Job Failures” table. Click “Get/Refresh Failed Job Metadata” to populate the table with details about your failed task. The `failures.causedBy.message` columns are of particular interest as they contain the error messages produced by the tool used within the task itself. In this example pictured below, I provided an erroneous path for one of my inputs, and as you can see, the tool “could not localize” it:
+
+![job_failures](/dasldemos/assets/proof_ts_job_failures.png)
+
+- If these messages don’t provide enough information, navigate back to the “Job List” table and take a look at a few additional columns:
+    - **stdout & stderr**: these columns contain the location of flat files that contain the output and error messages from the command line tools used during the task(s) in question. These will help to provide tool-specific feedback as to what might be going wrong. To access these, copy the path provided, log on to Rhino, and view the text files via `vi` , `vim`, `cat` , or some other similar text viewing tool.
```

Comment:
I'd suggest more, less, and cat. Those are easier for noobs and the suggestion of a text editor implies we may want users to edit the files.

Reply:
Great call, updated this to just say "`more`, `less`, or `cat`."